src/builders/thermodynamic.jl

```diff
@@ -0,0 +1,258 @@
+"""
+$(TYPEDSIGNATURES)
+
+The function uses the supplied `optimizer` and
+`reaction_standard_gibbs_free_energies`. Optionally, `flux_solution` can be used
+to set the directions of each reaction in `model` (all reactions are assumed to
+proceed forward and are active by default). The supplied `flux_solution` should
+be free of internal cycles i.e. thermodynamically consistent. This optional
+input is important if a reaction in `model` normally runs in reverse (negative
+flux). Note, reactions in `flux_solution` that are smaller than `small_flux_tol`
+are also ignored in the analysis function (for numerical stability).
+
+
```

Comment:
double nl?

Reply:
ah, the docs are out of date, upding them now